manuscript/manuscript.md

```diff
@@ -311,13 +310,17 @@ their grant guidelines to prevent sharing personally identifiable information,
 and to anonymize data on human subjects.
 
 In small datasets, a hashing scheme (anonymizing PII by converting it into
-a numeric key of a fixed length with a standard alogrithm) is enough to
+a numeric key of a fixed length with a standard algorithm) is enough to
 anonymize minimal personal information. Make sure to not store the hashing scheme
-with the data to prevent inadvertent sharing and don't use a commonplace hashing
+with the data to prevent inadvertent sharing and to not use a commonplace hashing
 technique. Famously, New York City officials shared what they thought was
 anonymized data on cab drivers and over 173 million cab rides. However, it was
 quickly recognized that the city anonymized the data with a simple MD5 hashing
 scheme and all 20 GB of data were de-anonymized in a matter of hours [@Goodin].
+This type of error can be prevented by asking a trusted colleague to try to
```

Comment:
I know little about security, but it seems unlikely that "asking a trusted colleague to try" could "prevent" something like this. It sounds from the article clear that the error was using that the data folk providers thought MD5 was sufficient to anonymize the data, but were wrong. That is a rule that can be followed.

If the issue is security, better to have a security expert review the data than a trusted colleague.

My suggestion: delete the sentence that starts on 320 and ends on 321; optionally select another example.

Also, is there a technical source or example other than Ars technica that we could use to make this point? If we use this example, would it be better to use a primary source like https://medium.com/@vijayp/of-taxis-and-rainbows-f6bc289679a1 than a news story?